src/invoices/InvoiceDetails/hooks/useHasPendingOrders.js

```diff
@@ -0,0 +1,33 @@
+import { useMemo } from 'react';
+import { ORDER_STATUSES } from '@folio/stripes-acq-components';
+import { useOrders } from '../../../common/hooks';
+
+export const useHasPendingOrders = (orderLinesMap = {}) => {
+  const orderIds = useMemo(() => {
+    return Object.values(orderLinesMap)
+      .map(({ purchaseOrderId }) => purchaseOrderId);
+  }, [orderLinesMap]);
+
+  const {
+    isLoading,
+    orders,
+    isFetched,
+  } = useOrders(orderIds);
+
+  const hasPendingOrders = useMemo(() => {
+    if (isLoading) {
+      return true;
+    }
+
+    if (isFetched && !orders.length) {
```

Comment:
I don't think you need it at all if you follow https://github.com/folio-org/ui-invoice/pull/694/files#r1212551416